gitevents.js

```diff
@@ -84,6 +86,18 @@ var issueHandler = function issueHandler(event) {
           }
 
         });
+=======
```

Comment:
Expected an identifier and instead saw '==='.<br>Expected an operator and instead saw '==='.<br>Expected an operator and instead saw '='.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.